buildkitd/buildkitd.go

```diff
@@ -934,6 +937,20 @@ func printBuildkitInfo(bkCons conslogging.ConsoleLogger, info *client.Info, work
 		default:
 		}
 	}
+
+	var cacheInfo *client.PruneInfo
+	for _, p := range workerInfo.GCPolicy {
```

Comment:
nit: move this logic into a helper func so we could do `cacheSize, ok := getGCPolicySize(workerInfo.GCPolicy)` rather than have to worry about `cacheInfo` being nil.

Reply:
Updated.

It's interesting how verbosity or DRY differ in Go versus other languages :slightly_smiling_face: